situp-core/integrationTest.gradle

```diff
@@ -75,9 +91,12 @@ task stopSitupDockerContainer(type: DockerStopContainer) {
  * ODFE Docker tasks
  */
 task createOdfeDockerContainer(type: DockerCreateContainer) {
+    dependsOn createSitupNetwork
```

Comment:
nit/question: does having two tasks that depend on creating the situp network mean this can run twice?

Reply:
From the build lifecycle document it is not the case: https://docs.gradle.org/current/userguide/tutorial_using_tasks.html